src/presolve/presolve_utils.jl

```diff
@@ -45,94 +45,93 @@ end
 find_empty_rowscols(v_cnt::Vector{Int}) = findall(isequal(0), v_cnt)
 find_singleton_rowscols(v_cnt::Vector{Int}) = findall(isequal(1), v_cnt)
 
-function remove_rowscols_A!(Arows, Acols, Avals, kept_rows, kept_cols, nvar, ncon)
-  Annz = length(Arows)
-  Arm = 0
-  nb_rm_rows = 0
-  # remove rows
-  for i0 = 1:ncon
-    kept_rows[i0] && continue
-    i = i0 - nb_rm_rows # up idx according to already removed rows
-    nb_rm_rows += 1
-    Awritepos = 1
-    k = 1
-    Arm_tmp = 0
-    while k <= Annz - Arm
-      Ai, Aj, Ax = Arows[k], Acols[k], Avals[k]
-      if Ai == i
-        Arm_tmp += 1
-      else
-        Arows[Awritepos] = (Ai < i) ? Ai : Ai - 1
-        Acols[Awritepos] = Aj
-        Avals[Awritepos] = Ax
-        Awritepos += 1
-      end
-      k += 1
+function SparseArrays.dropzeros!(A::SparseMatrixCOO{T}) where {T}
```

Comment:
Waiting for https://github.com/JuliaSmoothOptimizers/SparseMatricesCOO.jl/pull/17 to remove this.